_spinnaker_install_admin_guides/install-on-aws-halyard-pod.md

```diff
@@ -0,0 +1,668 @@
+11. Click "Close"
+
+Then, add an inline policy to your IAM user:
+
+1. Click on your newly-created IAM user
+2. Click on "Add inline policy" (on the right)
+3. Click on the "JSON" tab
+4. Add this text (replace `s3-spinnaker-abcxyz` with the name of your bucket)
+
+	```json
+	{
+	  "Version": "2012-10-17",
+	  "Statement": [
+	    {
+	      "Effect": "Allow",
+	      "Action": "s3:*",
+	      "Resource": [
+	        "arn:aws:s3:::spinnaker-abcxyz",
+	        "arn:aws:s3:::spinnaker-abcxyz/*"
+	      ]
+	    }
+	  ]
+	}
+	```
+
+5. Click on "Review Policy"
+
+6. Give your inline policy some name. For example `s3-spinnaker-abcxyz`
+
+7. Click "Create Policy"
+
+## Create an IAM policy attached to the Kubernetes nodes, using an inline policy
+
+Alternately, you can attach an IAM policy to the role attached to your Kubernetes nodes.
+
+1. Log into the AWS Console (Web UI)
+2. Navigate to EC2 (Click on "Services" at the top, and then on "EC2" under "Compute")
+3. Click on one of your Kubernetes nodes
+4. In the bottom section, look for "IAM role" and click on the role
+5. Click on "Add inline policy" (on the right)
+6. Click on the "JSON" tab
+7. Add this text (replace `s3-spinnaker-abcxyz` with the name of your bucket)
+
+	```json
+	{
+	  "Version": "2012-10-17",
+	  "Statement": [
+	    {
+	      "Effect": "Allow",
+	      "Action": "s3:*",
+	      "Resource": [
+	        "arn:aws:s3:::spinnaker-abcxyz",
+	        "arn:aws:s3:::spinnaker-abcxyz/*"
+	      ]
+	    }
+	  ]
+	}
+	```
+
+8. Click on "Review Policy"
+
+9. Give your inline policy some name. For example `s3-spinnaker-abcxyz`
+
+10. Click "Create Policy"
+
+# Start the Halyard container
+
+Halyard is a tool used to install Spinnaker (it generates Kubernetes manifests for each of the Spinnaker services).  First, create a namespace for Spinnaker to run in:
+
+```bash
+# Feel free to use a different namespace for this
+kubectl create ns spinnaker
+```
+You can use this manifest to install and run Halyard (call the file `halyard.yml`)
+
+```yml
+# halyard.yml
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: spinnaker
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- 
+  kind: ServiceAccount
+  name: default
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: hal-pvc
+  labels:
+    app: halyard
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 100Mi
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: halyard
+spec:
+  replicas: 1
+  serviceName: halyard
+  selector:
+    matchLabels:
+      app: halyard
+  template:
+    metadata:
+      labels:
+        app: halyard
+    spec:
+      containers:
+      - 
+        name: halyard
+        image: armory/halyard-armory:{{ site.data.versions.halyard-armory-version }}
+        volumeMounts:
+        - 
+          name: hal
+          mountPath: /home/spinnaker/.hal
```

Comment:
still prefer mounting /home/spinnaker 
because things like .aws and .kube 
and being able to save .bashrc files just make life easier.

Reply:
@away168 the instructions assume you're running in a cluster not local on your machine, I believe, so can't mount the local /home/spinnaker if that is what you're saying?

Reply:
i made this change, but i wanna do a bit more testing before we publish.

Reply:
> @away168 the instructions assume you're running in a cluster not local on your machine, I believe, so can't mount the local /home/spinnaker if that is what you're saying?

I meant for the VPC to be mounted to /home/spinnaker as opposed to /home/spinnaker/.hal
since other useful things reside in the home directory, and i'd like those things to persist if halyard gets restarted or updated.